Installation/CMakeLists.txt

```diff
@@ -108,6 +108,11 @@ else ( CGAL_BRANCH_BUILD )
 
   # Enable testing with BUILD_TESTING
   option(BUILD_TESTING "Build the testing tree." OFF)
+  #enable coverage with TESTING_WITH_COVERAGE
+  option(TESTING_WITH_COVERAGE "Build the testing tree with coverage." OFF)
+  if(TESTING_WITH_COVERAGE)
+    set(BUILD_TESTING ON)
+  endif(TESTING_WITH_COVERAGE)
```

Comment:
That modification is probably not wanted, and a leftover of your other work on covrage.